src/app/public/modules/href/href-resolver.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { TestBed } from '@angular/core/testing';
+import { HrefResolverFixtureService } from './fixtures/href-resolver-fixture.service';
+import { SkyHrefResolverService } from './href-resolver.service';
+
+describe('HREF Resolver Service', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        {
+          provide: SkyHrefResolverService,
+          useClass: HrefResolverFixtureService
+        }
+      ]
+    });
+  });
+
+  it('should resolve an href through fixture', () => {
+    const fixtureService = new HrefResolverFixtureService();
```

Comment:
Since you remove the multi-provider functionality, these unit tests should just test the base functionality of `SkyHrefResolverService`, which is to pass back the URL that's passed in. There's no need to test the fixtures service since that's really only useful for testing the directive.